src/Common/ZooKeeper/ZooKeeperCommon.cpp

```diff
@@ -1004,7 +1004,10 @@ void ZooKeeperSessionIDRequest::readImpl(ReadBuffer & in)
 
 Coordination::ZooKeeperResponsePtr ZooKeeperSessionIDRequest::makeResponse() const
 {
-    return std::make_shared<ZooKeeperSessionIDResponse>();
+    auto response = std::make_shared<ZooKeeperSessionIDResponse>();
```

Comment:
let's use it here https://github.com/ClickHouse/ClickHouse/blob/master/src/Coordination/KeeperStateMachine.cpp#L567-L569